src/data/layer-2/cex-layer-2-support.json

```diff
@@ -0,0 +1,32 @@
+[
+  {
+    "name": "Binance",
+    "supports_withdrawals": ["Arbitrum"],
+    "supports_deposits": ["Arbitrum"],
+    "url": "https://www.binance.com/"
+  },
+  {
+    "name": "Crypto.com",
+    "supports_withdrawals": ["Arbitrum"],
+    "supports_deposits": ["Arbitrum"],
+    "url": "https://crypto.com/"
+  },
+  {
+    "name": "OKX",
+    "supports_withdrawals": ["Arbitrum"],
+    "supports_deposits": ["Arbitrum"],
+    "url": "https://www.okx.com/"
+  },
+  {
+    "name": "FTX",
+    "supports_withdrawals": ["Arbitrum"],
+    "supports_deposits": ["Arbitrum"],
+    "url": "https://ftx.com/"
+  },
+  {
+    "name": "Huobi",
+    "supports_withdrawals": ["Arbitrum"],
```

Comment:
Huobi also supports withdrawals and deposits into Optimism. https://www.huobi.com/support/en-us/detail/24902339096439?invite_code=vdcu7

Reply:
Fixed! Thanks!